app/routes/events.tsx

```diff
@@ -1,6 +1,8 @@
 import dayjs from 'dayjs';
```

Comment:
are we keeping the events static as of now?
for both `events` & `socialEvents` routes?

Reply:
Currently yes, when we have clear private, public and social events are created we are replace accordingly

Reply:
makes sense